rpi0w-nexmon.sh

```diff
@@ -350,20 +317,20 @@ mkdir -p ${basedir}/root/scripts
 wget https://raw.github.com/dweeber/rpiwiggle/master/rpi-wiggle -O ${basedir}/root/scripts/rpi-wiggle.sh
 chmod 755 ${basedir}/root/scripts/rpi-wiggle.sh
 
-# Firmware needed for rpi3 wifi (default to standard aka not nexmon)
+# Firmware needed for rpi3 wifi (copy nexmon firmware)
 mkdir -p ${basedir}/root/lib/firmware/brcm/
+cp ${basedir}/../misc/rpi3/brcmfmac43430-sdio-nexmon.bin ${basedir}/root/lib/firmware/brcm/brcmfmac43430-sdio.bin
 cp ${basedir}/../misc/rpi3/brcmfmac43430-sdio.txt ${basedir}/root/lib/firmware/brcm/
-cp ${basedir}/../misc/rpi3/brcmfmac43430-sdio.bin ${basedir}/root/lib/firmware/brcm/
-
-# Copy firmware for original backup for Nexmon
-cp ${basedir}/../misc/rpi3/brcmfmac43430-sdio.txt ${basedir}/root/opt/nexmon/firmware/brcmfmac43430-sdio.txt
-cp ${basedir}/../misc/rpi3/brcmfmac43430-sdio.bin ${basedir}/root/opt/nexmon/firmware/brcmfmac43430-sdio.orig.bin
 
-# Copy nexmon firmware to /opt/nexmon/firmware folder
-cp ${basedir}/../misc/rpi3/brcmfmac43430-sdio-nexmon.bin ${basedir}/root/opt/nexmon/firmware/brcmfmac43430-sdio.bin
+# Copy nexutil
+cp ${basedir}/../misc/rpi3/nexutil-pi0 ${basedir}/root/usr/bin/nexutil
+chmod +x ${basedir}/root/usr/bin/nexutil
 
 cd ${basedir}
 
+# Copy Kali bashrc to root
+cp ${basedir}/../misc/bashrc ${basedir}/root/.bashrc
```

Comment:
cp ${basedir}/root/etc/bash.bashrc ${basedir}/root/root/.bashrc - there's no reason to add the bashrc into the repository when it's installed by a package.  Also, ${basedir}/root is the rootfs mountpoint, not the root home directory :)